src/services/utilities.ts

```diff
@@ -179,6 +179,7 @@ namespace ts {
 
         switch (node.parent.kind) {
             case SyntaxKind.TypeReference:
+            case SyntaxKind.ImportType:

```

Comment:
This should probably only be true if `!(node.parent as ImportTypeNode).isTypeOf`, otherwise it acts like a type query instead.

Reply:
We do normally consider `typeof` uses to be references though, e.g.:
```ts
const [|x|] = 0;
const y: typeof [|x|] = 0;
```

Reply:
Will the results still be classified correctly if that's picked up here and not where type queries are looked at?

Reply:
There actually isn't a specific place in find-all-references where we look for type queries -- we basically just look for identifiers with the same name and check for a symbol match.

Reply:
But this function is called from ```getMeaningFromLocation``` so that means we need to handle this irrespectively to avoid any wrong usage being seeping out.